README.md

```diff
@@ -19,7 +23,10 @@ The GitHub repository for Homebrew core.
     homebrew_prefix: /usr/local
     homebrew_install_path: "{{ homebrew_prefix }}/Homebrew"
 
-The path where Homebrew will be installed (`homebrew_prefix` is the parent directory). It is recommended you stick to the default, otherwise Homebrew might have some weird issues. If you change this variable, you should also manually create a symlink back to /usr/local so things work as Homebrew expects.
+The path where Homebrew will be installed (`homebrew_prefix` is the parent
```

Comment:
One note on these changes—I typically use one-line-per-paragraph in my docs, as word wrapping is easier in editors than trying to manage text changes while breaking at 80 cols. For code, I stick to 80 col, but for docs, I like to leave it free-flowing, just because docs change a lot :)